docs/apm/deployment-annotations.asciidoc

```diff
@@ -7,13 +7,40 @@
 ++++
 
 For enhanced visibility into your deployments, we offer deployment annotations on all transaction charts.
-This feature automatically tags new deployments, so you can easily see if your deploy has increased response times
-for an end-user, or if the memory/CPU footprint of your application has changed.
-Being able to identify bad deployments quickly enables you to rollback and fix issues without causing costly outages.
+This feature enables you to easily determine if your deploy has increased response times for an end-user,
+or if the memory/CPU footprint of your application has changed.
+Being able to quickly identify bad deployments enables you to rollback and fix issues without causing costly outages.
+
+By default, automatic deployment annotations are enabled.
```

Comment:
@dgieselaar Can automatic deployment annotations be disabled? If so, how?

Reply:
They can't _really_ be disabled. The only thing that you can do to have them not show up is either by not setting `service.version`, or storing an annotation via the API, and then derived annotations will still be displayed for date ranges without a stored annotation.